main.nf

```diff
@@ -144,6 +145,8 @@ if( params.bwa_index ){
     bwa_index = Channel
         .fromPath(params.bwa_index)
         .ifEmpty { exit 1, "BWA index not found: ${params.bwa_index}" }
+    bwa_index_genomesizetable = Channel
```

Comment:
This can be done in a neater way when combined with the above:

```nextflow
Channel.fromPath(params.bwa_index)
    .ifEmpty { exit 1, "BWA index not found: ${params.bwa_index}" }
    .into {bwa_index, bwa_index_genomesizetable}
```

Reply:
Yes, this was my initial method. But then I got problem that the validation of samples in MACS config failed, which was exactly the same case when I tested moving GTF and BED into channels. To be honest I don't really understand the cause. But the current method works.

Reply:
But these are both `Channel.fromPath`, not `file()` - they should be identical to one another?

Reply:
Yes, I was also confused. But once I splitted the channel into multiple outputs I ran into that error...A lot of struggle there.